content/en/docs/ops/configuration/mesh/config-resource-ready/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e  # Exit on failure
+set -u  # Unset is an error
+# There is no need to echo, output appears in TestDocs/ops/configuration/mesh/config-resource-ready/test.sh/test.sh/_test_context/test.sh_debug.txt
+set -o pipefail
+
+# This script doesn't need a control plane initially and will install Istio when needed
+# @setup profile=none
+
+echo '*** config-resource-ready step 1 ***'
+echo y | snip_install_with_enable_status
+
+echo '*** istioctl-analyze step 2 ***'
+_verify_contains snip_apply_and_wait_for_httpbin_vs "$snip_apply_and_wait_for_httpbin_vs_out"
+
+
+# @cleanup
+kubectl delete -f samples/httpbin/httpbin.yaml || true
```

Comment:
Hmm, why do you have `|| true` here? It shouldn't be needed since the framework automatically adds a `set +e` command at the beginning of the cleanup script: https://github.com/istio/istio.io/blob/47c8caa3090d5c81d331d2396776d259cd786ed2/pkg/test/istioio/framework.go#L63